### PR TITLE
Add polish language

### PR DESCRIPTION
**Pull request by Triloworld**
New language for status page - Polish (PL)

### CODE REVIEW

**Comment by Triloworld** [comment]
I add Polish to interface. Added locale - https://github.com/moment/moment/blob/develop/locale/pl.js

**Comment by Triloworld** [comment]
If you can check and merge then will be nice improve :)
@dungphanxuan @pschlan

**Comment by Triloworld** [comment]
I can't find a way to translate website - is this some different repository

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (pschlan) [inline .prettierrc:1]: Can you remove this file from the PR? Thanks

File: statuspage/src/locales/pl/translation.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "status": {
+    "availability": "Dostępność",
+    "availabilityWarning": "Availability warning",
+    "availabilityError": "Availability error",
+    "lowAvailabilityText": "We're currently experiencing reduced availability of {{percent}} % in the last 30 minutes.",
+    "latency": "Opóźnienie",
+    "latencyWarning": "Latency warning",
+    "latencyError": "Latency error",
+    "highLatencyText": "We're currently experiencing unusually high latency of {{ms}} ms in the last 30 minutes.",
```
Comment (pschlan) [inline statuspage/src/locales/pl/translation.json:21]: Any chance to also get a translation for these? :)

**Review by pschlan — COMMENTED** [review]
Awesome, thanks so much, just a few minor comments.

**Comment by pschlan** [comment]
> I can't find a way to translate website - is this some different repository

We don't currently have this on GitHub. When I find some time, I will clean it up and publish it, but no ETA yet.

File: .prettierrc (hunk 1 above)
Comment (Triloworld) [reply .prettierrc:1]: Sorry for that. It is for disabling styles from plugin to not destroy how is done

File: statuspage/src/locales/pl/translation.json (hunk 2 above)
Comment (Triloworld) [reply statuspage/src/locales/pl/translation.json]: Done. Thx

**Comment by Triloworld** [comment]
It is success in deployment. Polish as language in dropdown don't show up - only this issue remaining.
Zrzut ekranu 2024-03-7 o 11 46 35
Zrzut ekranu 2024-03-7 o 11 49 09
All other elements look great. Thanks! I will wait for production deployment.

**Pull request merged**